README.rst

```diff
@@ -320,6 +320,19 @@ or edited by other administrators. Only the superuser has the permissions to do
 
 By default, the first manager of an organization is designated as the owner of that organization.
 
+Organization Users
+------------------
+
+Deleting Organization Users
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Becuase of the ``CASCADE`` feature of django, deleting an organization user should
+consequently delete the Organization Owner which depends on it. Doing this may lead
+to confusion as users may delete owners inadvertently. Becuase of this, users are not
+allowed to delete an organization user which belongs to an owner. If they really
+want to do this, then they will have to delete its organization owner first before
+deleting the organization user.
```

Comment:
I find this confusing. It's not about CASCADE that we're adding this.
I would just keep it simple, is not important, is a corner case, either we should explain it briefly in two lines (but correctly) or remove it.